src/CMakeLists.txt

```diff
@@ -60,19 +60,19 @@ include_directories(SYSTEM
 	${FFTW3F_INCLUDE_DIRS}
 )
 
-IF(NOT ("${PULSEAUDIO_INCLUDE_DIR}" STREQUAL ""))
+IF(PULSEAUDIO_INCLUDE_DIR)
 	include_directories(SYSTEM "${PULSEAUDIO_INCLUDE_DIR}")
 ENDIF()
 
-IF(NOT ("${LV2_INCLUDE_DIRS}" STREQUAL ""))
+IF(LV2_INCLUDE_DIRS)
 	include_directories(SYSTEM ${LV2_INCLUDE_DIRS})
 ENDIF()
 
-IF(NOT ("${LILV_INCLUDE_DIRS}" STREQUAL ""))
-	include_directories(SYSTEM ${LILV_INCLUDE_DIRS})
+IF(Lilv_INCLUDE_DIRS)
```

Comment:
I believe this will cause issues with plugins/Lv2* because they are expecting `LV2_INCLUDE_DIRS` too, no?

Reply:
Whoops, I meant to type `LILV_INCLUDE_DIRS`.  Same question applies. Corrected.